cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -808,27 +808,41 @@ def _command_runner(
         Returns:
             dict(): command execution result if `log_record` is defined else None
         """
+        response = None
+        need_check_server_status = False
         if command.action == "ssh_command":
-            return self._command_runner_ssh(
+            response = self._command_runner_ssh(
                 log_record,
                 rendered_command_code,
                 rendered_command_path,
                 ssh_connection,
                 **kwargs,
             )
+            need_check_server_status = True
         elif command.action == "file_using_template":
-            return self._command_runner_file_using_template(
+            response = self._command_runner_file_using_template(
                 log_record,
                 command.file_template_id,
                 rendered_command_path,
                 **kwargs,
             )
+            need_check_server_status = True
         elif command.action == "python_code":
-            return self._command_runner_python_code(
+            response = self._command_runner_python_code(
                 log_record,
                 rendered_command_code,
                 **kwargs,
             )
+            need_check_server_status = True
+
+        if need_check_server_status:
```

Comment:
Please, don't copy need_check_server_status variable, use `else` operator to set default value
 
```python
        response = None
        need_check_server_status = True

        if command.action == "ssh_command":
            response = self._command_runner_ssh(
                log_record,
                rendered_command_code,
                rendered_command_path,
                ssh_connection,
                **kwargs,
            )
        elif command.action == "file_using_template":
            response = self._command_runner_file_using_template(
                log_record,
                command.file_template_id,
                rendered_command_path,
                **kwargs,
            )
        elif command.action == "python_code":
            response = self._command_runner_python_code(
                log_record,
                rendered_command_code,
                **kwargs,
            )
        else:
            need_check_server_status = False

```